features/support/capybara.rb

```diff
@@ -0,0 +1,18 @@
+Capybara.register_driver :chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w[window-size=1280,960], w3c: false }
```

Comment:
do we need to turn off `w3c`? what does it do?

Reply:
Since we're using Selenium 3 and not 4 I assume that's still the case:

https://medium.com/@juanba48/selenium-4-is-now-w3c-compliant-what-does-this-mean-ceb44de2d29b